content/download/fedora.adoc

```diff
@@ -4,14 +4,24 @@ iconhtml = "<div class='fl-fedora'></div>"
 weight = 20
 +++
 
-=== 4.0.2 Stable Release
-KiCad 4.0.2 is available in the Fedora 23 repo.
+=== Stable Release
+In general, latest Fedora releases ship the stable versions KiCad as they are
+released, with a delay of around a week that the package spends in the testing
```

Comment:
If you meant "we should *remove* the info" then I don't object removing it really. My point was to make sure the user is not too surprised if they see 4.0.5 when 4.0.6 was already released. But perhaps this is really not necessary. Will follow up. with an updated patch.